pkg/metrics/metrics.go

```diff
@@ -102,6 +108,17 @@ The label 'result' can have the following values:
 		}, []string{"cluster_queue", "status"},
 	)
 
+	LocalQueuePendingWorkloads = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: constants.KueueName,
+			Name:      "local_queue_pending_workloads",
+			Help: `The number of pending workloads, per 'local_queue' and 'status'.
+'status' can have the following values:
+- "active" means that the workloads are in the admission queue.
+- "inadmissible" means there was a failed admission attempt for these workloads and they won't be retried until cluster conditions, which could make this workload admissible, change`,
+		}, []string{"local_queue", "namespace", "status"},
```

Comment:
I think this is acceptable, but let me consider other options:
1. "local_queue", "namespace" - as in the proposal
2. "name", "namespace"
3. "local_queue" - key "namespace/name"

I'm not in favor of (3) because maybe for some use-cases one wants to aggregate metrics by LQ name rather than full key. 

My only slight preference for (2.) is that it is less redundant. It is already clear from the metrics name that we are talking LQs. This is not the case for the `pending_workloads` metrics for CQs, so I think we don't need to follow the naming pattern for params strictly here. WDYT?

Reply:
I'm happy with `2`